Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutliner.qss

```diff
@@ -12,12 +12,34 @@ AzToolsFramework--EntityOutlinerWidget #m_display_options
     qproperty-iconSize: 16px 16px;
 }
 
+#OutlinerModeSwitcherContainer
+{
+    background: #363636;
+    margin: 4px 0;
+    padding: 0;
+}
+
+#OutlinerModeSwitcher
+{
+    margin: 0 0 0 23px;
+    padding: 6px 0;
+}
+
+#OutlinerModeSwitcherHelp
+{
+    margin-right: 6px;
+}
+
+#OutlinerModeSwitcher QAbstractButton:checked
```

Comment:
I cannot find these being used anywhere in this PR or in existing code. Can be deleted?

Reply:
If they are not used anywhere in the code, I don't see any point in keeping them around. We shouldn't be merging in unused code when we aren't sure what it does.